code/modules/clothing/modular_armor/attachments/leg_plates.dm

```diff
@@ -48,3 +48,8 @@
 	name = "\improper Jaeger Pattern Ranger leg plates"
 	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Ranger armor piece."
 	greyscale_config = /datum/greyscale_config/armor_mk1/ranger
+
+/obj/item/armor_module/armor/legs/marine/trooper
+	name = "\improper Jaeger Pattern Trooper leg plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides high protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Trooper armor piece."
```

Comment:
Someone should probably update these given that all arm/leg pieces have the same armor and slowdown now.